less/admin/ExtensionWidget.less

```diff
@@ -3,8 +3,8 @@
   padding: 0;
 }
 
-.ExtensionsWidget-list {
-  > .container {
+.ExtensionsWidget {
+  .ExtensionsWidget-list {
```

Comment:
wouldn't just `.ExtensionWidget-list` be enough ? it's good practice to avoid over specification, to make customization easier for both admins and theme designers.